state-indexer/src/configs.rs

```diff
@@ -298,12 +303,25 @@ impl ScyllaStorageManager for ScyllaDBManager {
         scylla_db_session
             .query(
                 "
-                CREATE TABLE IF NOT EXISTS chunks (
-                    block_height varint,
+                CREATE TABLE IF NOT EXISTS blocks (
                     block_hash varchar,
+                    block_height varint,
+                    PRIMARY KEY (block_hash)
+                )
+                ",
+                &[],
+            )
+            .await?;
+
+        scylla_db_session
+            .query(
+                "
+                CREATE TABLE IF NOT EXISTS chunks (
                     chunk_hash varchar,
+                    block_height varint,
                     shard_id varint,
-                    PRIMARY KEY (chunk_hash)
+                    height_included varint,
```

Comment:
What's the difference between block_height and height_included?

Reply:
Ah, OK
If the chunk is duplicated, we will have
```
block_height height_included
10           10
10           11
10           12
```
Right?

Reply:
> Right?

Exactly!

Reply:
It's non-intuitive.
Let's rename the columns to help future us understand this.

Reply:
> height_included is actually included_in_block_height
> 
> block_height could be first_appeared_at_ block_height (don't like this, but I don't have any better ideas for now)

For me, table name + field name `chunks.height_included` is not worse than `chunks.included_in_block_height`. And looks obvious.

`first_appeared_at_block_height` that'd be misleading for your example 

first_appeared_at_block_height | included_in_block_height |
|---|--|
| 10 | 10 |
| 10 | 11 |
| 10 | 12 |

I have looked at your example and realized there is a mistake. It should be like this:

| block_height | height_included |
|---|--|
| 10 | 10 |
| 11 | 10 |
| 11 | 11 |

---

I don't think the suggested naming is better than the existing one.

Reply:
> And looks obvious.

I disagree. height_included what? Is height itself included in chunk? (no, I know this because I have domain knowledge. New team member will be doomed here)

Reply:
Can you extend your example to 3 blocks or N blocks? Will we have C(N, 2) rows? Why do we need to store it this way?

Reply:
included_in_block_height actually solves this concrete issue, it explicitly says that chuck is included in some block height.

Reply:
> included_in_block_height actually solves this concrete issue

OK, I am ready to accept it. What about the `block_height` then?

Reply:
Some questions are still unanswered:
Can you extend your example to 3 blocks or N blocks? Will we have C(N, 2) rows? Why do we need to store it this way?

Also, can you explain what `block_height` should be used for? I can't explain this, as a result, obviously, I can't name this.

Reply:
@telezhnaya and I had a call. Recap:

`block_height` identifies which block the given chunk appears in (it might appear in a couple of blocks, though).

We agreed to rename `height_included` into `stored_at_block_height`. This field points to the block height where the chunk can be found on the NEAR Lake S3 bucket (since there is only one copy of the chunk).

Whenever a chunk is missed and `nearcore` includes the previously available, we store this as a duplicate record, but `stored_at_block_height` will always point to the original source.